packages/twenty-front/src/modules/auth/sign-in-up/components/SignInUpForm.tsx

```diff
@@ -41,6 +44,13 @@ const StyledInputContainer = styled.div`
   margin-bottom: ${({ theme }) => theme.spacing(3)};
 `;
 
+const StyledForgotPasswordButton = styled(Button)`
```

Comment:
@i-am-chitti We should try to re-use existing UI components here.
I've looked at the existing Links we have and the closest one is the GithubVersionLink:
<img width="729" alt="image" src="https://github.com/twentyhq/twenty/assets/12035771/4f9adb09-2bc5-488f-a75a-015439deca5b">

You can play with it there: https://main--647862655ef7071d35328bb2.chromatic.com/?path=/story/ui-navigation-link-githubversionlink--default

@Bonapara should we re-use this one, and if yes how should we rename it?